### ISSUE TITLE
Binary Extraction doesn't save any data

### ISSUE THREAD

**Issue by dzid26** [gcarmix/HexWalk#32]
After pressing Extract All, the processing successfully complets, but it's not clear where the data is saved.
![image](https://github.com/gcarmix/HexWalk/assets/841061/7473ead7-0950-47f9-af75-f2f1150d787a)
I tried to use `binw.py -e` and it also doesn't say output anything, so maybe that is the root cause.

**Comment by gcarmix** [comment]
It seems a problem of binwalk and not of HexWalk, could you try the same file on Linux OS?

**Comment by dzid26** [comment]
> It seems a problem of binwalk and not of HexWalk, could you try the same file on Linux OS?

Yes, same thing on Ubuntu. Unless it saves the file in some mysterious location. I am attaching the binary for the reference 
[splash.zip](https://github.com/gcarmix/HexWalk/files/14550713/splash.zip)

I tried also:
```
python .\binw.py -e "C:\Apps\edl\C3\SOM1_clean_DUMP_rl\lun4\splash.bin"

DECIMAL       HEXADECIMAL     DESCRIPTION
--------------------------------------------------------------------------------
16384         0x4000          PC bitmap, Windows 3.x format,, 1080 x 2160 x 24
``` 
but there is no file output. 
Same on linux.

**Comment by dzid26** [comment]
The following kinda worked:
```
binwalk -D=".*" splash.bin
```

After changing extracted file to bmp, the bitmap was visible.

**Comment by dzid26** [comment]
> Ok, this is an issue of binwalk, in case you should open an issue here: https://github.com/ReFirmLabs/binwalk

Agree, but in this case wouldn't be better to use `binwalk -D=".*" ` arguments ?  
https://stackoverflow.com/a/43738351/1531161
https://stackoverflow.com/a/36635022/1531161

`-e`  never worked for me.  

Btw, binwalk seems like a [dead project](https://github.com/ReFirmLabs/binwalk/pulls) for now unfortunately.
